tests/RobotFramework/tests/tedge_write.robot

```diff
@@ -3,18 +3,52 @@ Resource            ../resources/common.resource
 Library             ThinEdgeIO
 
 Documentation
+...    This suite aims to check the correctness of tedge-write component which is used for privilege
+...    elevation to allow tedge processes running under `tedge` user to write to configuration files
+...    on the system tedge normally doesn't have permissions for.
+...
+...    tedge-write must:
+...    - write to files `tedge` have no permissions to (privilege elevation happens)
+...    - preserve permissions for existing files and apply new permissions for files created
+...    - handle permissions correcly on non-standard umasks
+...    - not leave temporary files
+...
 ...    Tests for tedge-write need to be done in RobotFramework because we need to run as a `tedge`
 ...    user, and a sudoers entry that allows `tedge` to run `sudo tedge-write` without password
 ...    needs to be present. Additionally, we need to ensure `tedge-write` works well on system where
 ...    the default umask is different.
+...
+...    Additionally, apart from the component itself, we need to test features that use it directly,
+...    like config_update.
 
 Suite Setup         Custom Setup
 Suite Teardown      Get Logs
 
 Test Tags           adapter:docker    theme:tedge-write
 
 
+*** Variables ***
+${REGULAR_USER} =    user
+
+
 *** Test Cases ***
+Sudo elevates permissions when tedge-write is run by tedge
```

Comment:
Sudo elevates permissions when tedge-write writes to a root-owned file
```suggestion
Sudo elevates permissions when tedge-write is run by tedge
```

Reply:
fixed in 66cdcc7fe